packages/trading-widget/src/trading-widget/components/withdraw/init-step/input-group/withdraw-section/withdraw-section.tsx

```diff
@@ -28,7 +28,7 @@ export const WithdrawSection: FC<WithdrawSectionProps> = (props) => {
                 size="sm"
                 className="!dtw-inline-block dtw-mb-0.5"
               />{' '}
-              {assetSymbol} withdraw includes 2 transactions
+              Withdrawing into {assetSymbol} includes 2 transactions
```

Comment:
```suggestion
              Withdrawing into {assetSymbol} involves 2 transactions:
```
missing colon.

should it be moved to `translation-provider.defaults.ts`?